acceptance/tests/fixtures/cases/api-gateways/jwt-auth/extraGatewayPolicy/extra-gateway-policy.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
idk if it is fine to nest things in here? didn't know where else to put this policy